tests/integration_tests/test_tracers.py

```diff
@@ -578,9 +579,9 @@ def test_refund_unused_gas_when_contract_tx_reverted(ethermint):
     assert "result" in tx_res
     tx_res = tx_res["result"]
     pre = int(tx_res["pre"][sender]["balance"], 16)
-    post = int(tx_res["post"][sender]["balance"], 16)
-    diff = pre - gas * gas_price * min_gas_multiplier - post
-    assert diff == 0, diff
```

Comment:
the original test compare the balance before and after the trace call and make sure that the diff is equals to `gas * gasprice *min gas multiplier` but now you are just comparing the balance of the sender before the trace call. which of course is not going to change the balance because its a request?

Reply:
the original test is correct, reverted